src/routes/api/likes/likes.routes.js

```diff
@@ -0,0 +1,17 @@
+import express from 'express';
+import LikesController from '../../../controllers/likes.controller';
+import validateToken from '../../../middlewares/auth';
+import confirmEmaiAuth from '../../../middlewares/emailVarification.middleware';
+import isLike from '../../../middlewares/likes.middleware';
+
+const router = express.Router();
+// greate edit a viewing profile handlers
+
+router.get('/claps/:Article', [validateToken, confirmEmaiAuth, isLike], LikesController.getClaps);
+router.get('/dislikes/:Article', [validateToken, confirmEmaiAuth, isLike], LikesController.getDislikes);
+router.put('/clap/:Article', [validateToken, confirmEmaiAuth, isLike], LikesController.clap);
```

Comment:
@salviosage I suggest we maintain the same naming convention for our routes ie `clap` to `claps`